src/docs/FgCardDocs.vue

```diff
@@ -1,60 +1,60 @@
 <template>
   <div>
-    <divider/>
+    <fg-divider/>
 
-    <header-large>
-      CARD
-    </header-large>
+    <fg-header-large>
+      FgCard
```

Comment:
I would make this header match the related label used in the side menu and therefore would drop the prefix:
```suggestion
      Card
```